src/name/publish.js

```diff
@@ -43,7 +43,7 @@ module.exports = (createCommon, options) => {
 
       const value = fixture.cid
 
-      ipfs.name.publish(value, (err, res) => {
+      ipfs.name.publish(value, { 'allow-offline': true }, (err, res) => {
```

Comment:
Camel case please - `allowOffline`.

Reply:
camel case doesn't work alan, this is used directly as http payload for go-ipfs

Reply:
It shouldn't be - core takes camel case options, if it needs to be converted to dash-case for the query string then `js-ipfs-http-client` needs to convert it and `js-ipfs` HTTP endpoint needs to convert it back. This is consistent with many of the other endpoints.

Reply:
@alanshaw can we defer this to another PR ? ill make an issue to track

Reply:
Sure, but we should get it resolved before the next release as it's a breaking change to a brand new feature otherwise.